Opserver/Content/js/Scripts.js

```diff
@@ -516,6 +516,44 @@ Status.Dashboard.Server = (function () {
                 5: { sorter: 'dataVal', sortInitialOrder: 'desc' }
             }
         });
+
+        function serviceAction(link, route, errMessage) {
+            var origText = link.text();
+            var $link = link.text('').prependWaveLoader();
+            var node = link.closest('[data-node]').data('node');
+            $.ajax(Status.options.rootPath + 'service/' + route, {
```

Comment:
Should be: `/dashboard/node/service/action`, and post the control action as a form variable - then we need a single route. Be sure to use the enum directly in the view as well, since we'll be matching it on the controller, this way you can find usages, etc. and everything lines up.